src/modules/shared/AddToCart/AddToCart.module.scss

```diff
@@ -17,6 +17,7 @@
   font-style: normal;
   font-weight: 700;
   line-height: 21px;
```

Comment:
Im not sure but maybe here we use our standart styles? If not font-style: normal; is redundant